src/test/resources/api_input/input2.json

```diff
@@ -0,0 +1,57 @@
+{
+  "dataCategories": [
+    {
+      "publishedArticles": [
+        { "urlName": "how-to-sign-in-using-a-computer", "title": "How to sign in using a computer", "id": "ka05I000000lBLzQAM", "dataCategories": [], "body": null },
+        { "urlName": "how-to-sign-in-using-a-mobile-tablet", "title": "How to sign in using a mobile/tablet", "id": "ka05I000000lBX8QAM", "dataCategories": [], "body": null },
+        { "urlName": "how-to-stay-signed-in", "title": "How to stay signed in", "id": "ka05I000000dOFAQA2", "dataCategories": [], "body": null },
+        { "urlName": "i-need-help-signing-in", "title": "I need help signing in", "id": "ka05I000000lBXDQA2", "dataCategories": [], "body": null },
+        { "urlName": "i-need-to-change-my-contact-details", "title": "I need to change my contact details", "id": "ka05I000000lBY1QAM", "dataCategories": [], "body": null },
+        { "urlName": "i-want-to-delete-my-account", "title": "I want to delete my account", "id": "ka05I000000UC52QAG", "dataCategories": [], "body": null },
+        { "urlName": "i-ve-forgotten-my-password", "title": "I've forgotten my password", "id": "ka05I000000UDqgQAG", "dataCategories": [], "body": null },
```

Comment:
tiny suggestion but keeping with consistency you could have `"ive-..."` like you have on line 39 instead of `"i-ve..."`

Reply:
yeah, that would be better to have as the URL for the article looks odd: https://manage.theguardian.com/help-centre/article/i-ve-forgotten-my-password

Unfortunately we can't change the urls after they've been published so will have to leave it as is.

Reply:
ah ok I hadn't realised that they already existed before this PR